kensu/utils/kensu.py

```diff
@@ -375,45 +401,69 @@ def report_with_mapping(self):
                     from_pks.add(from_guid)
                     schemas_pk.add(to_guid)
 
+
                 lineage = ProcessLineage(name=self.get_lineage_name(dataflow),
                                          operation_logic='APPEND',
                                          pk=ProcessLineagePK(
                                              process_ref=ProcessRef(by_guid=self.process.to_guid()),
                                              data_flow=dataflow))._report()
+                if lineage.to_guid() not in self.sent_runs:
```

Comment:
hmm in fact `self.sent_runs` variable stores the lineage ids? do we need to rename it to `sent_lineages`?